src/app/space/add-space-overlay/add-space-overlay.component.ts

```diff
@@ -25,34 +25,45 @@ import { SpacesService } from 'app/shared/spaces.service';
 })
 export class AddSpaceOverlayComponent implements OnInit {
   currentSpace: Space;
-  selectedTemplate: ProcessTemplate;
+  selectedTemplate: ProcessTemplate = null;
   spaceTemplates: ProcessTemplate[];
   space: Space;
 
   constructor(private router: Router,
-              public dummy: DummyService,
               private spaceService: SpaceService,
               private notifications: Notifications,
               private userService: UserService,
               private spaceNamespaceService: SpaceNamespaceService,
               private spaceNamePipe: SpaceNamePipe,
               private spacesService: SpacesService,
+              private spcaeTemplateService: SpaceTemplateService,
               private context: ContextService,
               private broadcaster: Broadcaster) {
-    this.spaceTemplates = dummy.processTemplates;
+    this.spaceTemplates = [];
     this.space = this.createTransientSpace();
   }
 
   ngOnInit() {
-    const srumTemplates = this.spaceTemplates.filter(template => template.name === 'Scenario Driven Planning');
-    if (srumTemplates && srumTemplates.length > 0) {
-      this.selectedTemplate = srumTemplates[0];
-    }
     this.context.current.subscribe((ctx: Context) => {
       if (ctx.space) {
         this.currentSpace = ctx.space;
       }
     });
+    this.spcaeTemplateService.getSpaceTemplates()
+      .subscribe((templates: ProcessTemplate[]) => {
+        this.spaceTemplates = templates.filter(t => t.attributes['can-construct']);
+        this.selectedTemplate = !!this.spaceTemplates.length ? this.spaceTemplates[0] : null;
+      }, () => {
+        console.log('Error on getting space templates');
```

Comment:
Please do not use console.log

Reply:
fixed in 269bc6e6789687cadcf3042c7bb63345a6beef63